From ef43dd51c97923f53c4c61e2051785077b9e5e30 Mon Sep 17 00:00:00 2001
From: Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
Date: Thu, 28 Mar 2019 13:47:16 -0400
Subject: [PATCH] Partially revert the change for enabling systemd-networkd.
 network-online.target may not be depended on by anything on minimal images,
 but multi-user.target already is.

---
 src/networkd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/networkd.c b/src/networkd.c
index 7c7ccef..909e580 100644
--- a/src/networkd.c
+++ b/src/networkd.c
@@ -824,7 +824,7 @@ cleanup_networkd_conf(const char* rootdir)
 void
 enable_networkd(const char* generator_dir)
 {
-    g_autofree char* link = g_build_path(G_DIR_SEPARATOR_S, generator_dir, "network-online.target.wants", "systemd-networkd.service", NULL);
+    g_autofree char* link = g_build_path(G_DIR_SEPARATOR_S, generator_dir, "multi-user.target.wants", "systemd-networkd.service", NULL);
     g_debug("We created networkd configuration, adding %s enablement symlink", link);
     safe_mkdir_p_dir(link);
     if (symlink("../systemd-networkd.service", link) < 0 && errno != EEXIST) {
-- 
2.20.1

